jupyterlab/lablauncher.sh

```diff
@@ -160,10 +160,23 @@ function setup_git() {
     fi
 }
 
+function change_stack_ownership() {
+    # This is really slow, so run it in the background.  Hopefully the user
+    #  isn't going to get settled enough to start installing things before it
+    #  finishes?  Not ideal, I'm afraid.
+    if [ -n "${U_NAME}" ]; then
+	local current_owner=$(ls -ld ${TOPDIR})
```

Comment:
I wasn't actually sure if, in bash, you could use an undeclared variable in a function then define it afterwards and resolve it. But it works. TIL.